rcl_yaml_param_parser/src/parser.c

```diff
@@ -1422,6 +1509,11 @@ static rcutils_ret_t parse_key(
           memmove((param_name + params_ns_len + 1U), value, param_name_len);
           param_name[tot_len - 1U] = '\0';
 
+          if (params_st->params[*node_idx].parameter_names[*parameter_idx] != NULL) {
```

Comment:
@brawner nit^2: invert operands order per style guide, as in
```suggestion
          if (NULL != params_st->params[*node_idx].parameter_names[*parameter_idx]) {
```

Reply:
Fixed.